### PR TITLE
feat: add emoji api

### PR DESCRIPTION
**Pull request by quanghuynguyen1902**
**What does this PR do?**

-   [x] add emojis api

- Get `api/v1/emojis?codes=eth,ftm`

### CODE REVIEW

File: pkg/routes/v1.go [hunk 1]
```diff
@@ -565,4 +565,10 @@ func NewRoutes(r *gin.Engine, h *handler.Handler, cfg config.Config) {
 	{
 		pkpassGroup.GET("", h.PkPass.GeneratePkPass)
 	}
+
+	emojisGroup := v1.Group("emojis")
```
Comment (trkhoi) [inline pkg/routes/v1.go:569]: /emojis

File: pkg/routes/v1.go [hunk 2]
```diff
@@ -565,4 +565,10 @@ func NewRoutes(r *gin.Engine, h *handler.Handler, cfg config.Config) {
...
+	{
+		emojisGroup.POST("list", h.Emojis.ListEmojis)
```
Comment (trkhoi) [inline pkg/routes/v1.go:571]: get all just need "/api/v1/emojis"
get list just need something like "/api/v1/emojis?query=ftm,eth"
I recommend create just 1 api and if no query param then return all, if yes then add query

**Review by trkhoi — CHANGES_REQUESTED** [review]

File: pkg/entities/emojis.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+	"github.com/defipod/mochi/pkg/logger"
+	"github.com/defipod/mochi/pkg/model"
+	"github.com/defipod/mochi/pkg/model/errors"
+)
+
+func (e *Entity) GetListEmojis(listCode []string) ([]*model.Emojis, error) {
+	emojis, err := e.repo.Emojis.GetListEmojis(listCode)
+	if err != nil {
+		e.log.Fields(logger.Fields{}).Error(err, "[entity.GetListEmojis] repo.emojis.GetListEmojis failed")
```
Comment (vhbien000) [inline pkg/entities/emojis.go:14]: Maybe we do not need Fields here 
```suggestion
		e.log.Error(err, "[entity.GetListEmojis] repo.emojis.GetListEmojis failed")
```

File: pkg/handler/emojis/emojis.go [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+func New(entities *entities.Entity, logger logger.Logger) IHandler {
+	return &Handler{
+		entities: entities,
+		log:      logger,
+	}
+}
+
+// ListEmojis   godoc
```
Comment (vhbien000) [inline pkg/handler/emojis/emojis.go:26]: Missing define request for swagger doc

File: pkg/repo/emojis/pg.go [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+func NewPG(db *gorm.DB) Store {
+	return &pg{db: db}
+}
+
+func (pg *pg) Create(emojis *model.Emojis) error {
+	return pg.db.Create(emojis).Error
+}
+
+func (pg *pg) GetListEmojis(listCode []string) (model []*model.Emojis, err error) {
+	return model, pg.db.Raw("SELECT id, code, discord_id, telegram_id, twitter_id FROM emojis WHERE code in (?)", listCode).Find(&model).Error
```
Comment (vhbien000) [inline pkg/repo/emojis/pg.go:22]: please check can we use following format, or sth like that
```suggestion
	return model, pg.db.Where("code in (?)", listCode).Find(&model)
```

File: pkg/repo/emojis/pg.go [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+func (pg *pg) GetAllEmojis() (model []*model.Emojis, err error) {
+	return model, pg.db.Raw("SELECT id, code, discord_id, telegram_id, twitter_id FROM emojis").Find(&model).Error
```
Comment (vhbien000) [inline pkg/repo/emojis/pg.go:26]: same above

File: pkg/routes/v1.go [hunk 7]
```diff
@@ -565,4 +565,9 @@ func NewRoutes(r *gin.Engine, h *handler.Handler, cfg config.Config) {
 	{
 		pkpassGroup.GET("", h.PkPass.GeneratePkPass)
 	}
+
+	emojisGroup := v1.Group("/emojis")
+	{
+		emojisGroup.GET("list", h.Emojis.ListEmojis)
```
Comment (trkhoi) [inline pkg/routes/v1.go:571]: emojisGroup.GET("/", h.Emojis.ListEmojis)

**Review by trkhoi — APPROVED** [review]

**Pull request merged**